backend/project_name/settings/base.py

```diff
@@ -132,11 +132,38 @@ def base_dir_join(*args):
 }
 
 # Celery
+# Recommended settings for reliability: https://gist.github.com/fjsj/da41321ac96cf28a96235cb20e7236f6
 CELERY_ACCEPT_CONTENT = ["json"]
 CELERY_TASK_SERIALIZER = "json"
 CELERY_RESULT_SERIALIZER = "json"
-CELERY_ACKS_LATE = True
+CELERY_TASK_ACKS_LATE = True
 CELERY_TIMEZONE = TIME_ZONE
+CELERY_BROKER_TRANSPORT_OPTIONS = {"confirm_publish": True, "confirm_timeout": 5.0}
+CELERY_BROKER_POOL_LIMIT = config("CELERY_BROKER_POOL_LIMIT", cast=int, default=1)
+CELERY_BROKER_CONNECTION_TIMEOUT = config(
+    "CELERY_BROKER_CONNECTION_TIMEOUT", cast=float, default=30.0
+)
+CELERY_REDIS_MAX_CONNECTIONS = config(
+    "CELERY_REDIS_MAX_CONNECTIONS", cast=lambda v: int(v) if v else None, default=None
```

Comment:
default=None causes the cast to be called? Because if so, we need to fix https://gist.github.com/fjsj/da41321ac96cf28a96235cb20e7236f6 too

Reply:
It raises the following:

`TypeError: int() argument must be a string, a bytes-like object or a real number, not 'NoneType'`

Reply:
I believe I don't have access to edit the gist, can you do it?

Reply:
I did it, thanks